README.md

```diff
@@ -86,6 +86,12 @@ When importing the code into Android Studio, use the **surface-duo-sdk** folder
     implementation "com.microsoft.device.dualscreen:navigation-ui-ktx:$nav_version"
     ```
 
+   **Testing**
+
+    ```gradle
+    implementation "com.microsoft.device.dualscreen.testing:testng-kotlin:1.0.0-alpha1"
```

Comment:
I think there is a typo here, missing an "i" in  `:testng-kotlin:`

Reply:
😱😱😱 good catch! thanks!